index.js

```diff
@@ -200,7 +200,9 @@ function _createXHR(options) {
             }
         }
     } else if (options.headers && !isEmpty(options.headers)) {
-        throw new Error("Headers cannot be set on an XDomainRequest object")
+        if (('ignoreHeadersWhenXDR' in options) == false || !!!options.ignoreHeadersWhenXDR) {
```

Comment:
`!!!` seems loud. is it any different from using just one? 
Also, `('ignoreHeadersWhenXDR' in options) == false` doesn't seem different from just `!options.ignoreHeadersWhenXDR`